pkg/kubelet/dockershim/convert.go

```diff
@@ -169,3 +170,18 @@ func toRuntimeAPISandbox(c *dockertypes.Container) (*runtimeapi.PodSandbox, erro
 		Annotations: annotations,
 	}, nil
 }
+
+func checkpointToRuntimeAPISandbox(id string, checkpoint *PodSandboxCheckpoint) *runtimeapi.PodSandbox {
+	state := runtimeapi.PodSandboxState_SANDBOX_NOTREADY
+	labels := make(map[string]string)
+	labels[kuberuntime.KubernetesManagedLabel] = "true"
```

Comment:
Labels are immutable, can we also checkpoint it?

And what about the other labels? Are we sure that kubelet will only use the `KubernetesManagedLabel` label?